docs/src/components/HeadingWithLink.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import styled from "styled-components";
+import { ButtonLink, Heading, Stack } from "@kiwicom/orbit-components";
+import { As, Type as HeadingType } from "@kiwicom/orbit-components/lib/Heading";
+import { Link as LinkIcon } from "@kiwicom/orbit-components/icons";
+
+import { slugify } from "../utils/common";
+
+export const StyledAnchorWrapper = styled.div`
+  margin-top: -10px;
+
+  svg {
+    opacity: 0%;
+    transition: opacity ${({ theme }) => theme.orbit.durationNormal} ease-in;
+  }
+
+  &:hover svg {
+    opacity: 100%;
+  }
+`;
+
+interface Props {
+  headingText: string;
+  headingLevel?: 2 | 3 | 4 | 5 | 6;
+  noId?: boolean;
+  spaceAfter?: "smallest" | "small" | "normal" | "medium" | "none";
```

Comment:
can you add export of `spaceAfter` type inside main package and use it here?

Reply:
It already was exported, so I just used it.